src/notifications/batterynotifier.cpp

```diff
@@ -153,6 +158,12 @@ void BatteryNotifier::prepareNotification()
             checkChargingTimer->start();
         }
     }
+    if (isStateChanged) {
+        if (!isAnyChargingState)
```

Comment:
Not a big deal either way, whichever you prefer :-)  Just thought I'd mention it.